AutoScout24/showcar-tracking
# Gt-127: Make GA PageView function synchronous

### Description
Make page tracking function synchronous

Save adBlockerStatus in the sessionStorage to make trackPageview() synchronous because PageView event should be fast and non-blocking as some events are implemented as PageView events. 

JIRA - https://autoscout24.atlassian.net/browse/GT-127

### Steps to Test Locally
* Run the service locally.
* Go to: http://local.autoscout24.de:3000/test.html
* Inspect DataLayer and find entry with "common_pageName" attribute.
* In the entry, verify that "adblocker_usage" attribute is set to "undefined" _(`undefined` is set only once per session)_.
* Refresh the page
* Verify that "adblocker_usage" attribute is set to "no adblock detected" or "adblock active".
